packages/roosterjs-editor-plugins/lib/plugins/HyperLink/HyperLink.ts

```diff
@@ -141,7 +141,6 @@ export default class HyperLink implements EditorPlugin {
 
                 let href: string | null;
                 if (
-                    !Browser.isFirefox &&
                     (href = this.tryGetHref(anchor)) &&
                     isCtrlOrMetaPressed(event.rawEvent) &&
                     event.rawEvent.button === 0
```

Comment:
If this behavior changed overtime and Firefox is no longer opening the links by default, should we prevent the default behavior of the event if browser is Firefox in case a user with an older version and tries to open a link with Ctrl + Click?

Reply:
True, I don't think this would have any side effects so might as well preventDefault it just in case